### PR TITLE
Add reference documentation for namespace selector

### PR DESCRIPTION
**Pull request by caseydavenport**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Hold merge until code is in master + v3.0

File: _includes/master/entityrule.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
-| tag                   | Deprecated: Match on tag. |  | string | |
-| notTag                | Deprecated: Negative match on tag. |  | string | |
 | nets                  | Match packets with IP in any of the listed CIDRs. | List of valid IPv4 or IPv6 CIDRs  | list of cidrs |
 | notNets               | Negative match on CIDRs. Match packets with IP not in any of the listed CIDRs. | List of valid IPv4 or IPv6 CIDRs  | list of cidrs |
 | selector    | Positive match on selected endpoints. | Valid selector | [selector](#selector) | |
 | notSelector | Negative match on selected endpoints. | Valid selector | [selector](#selector) | |
+| namespaceSelector | Positive match on selected namespaces. If specified, only workload endpoints in the selected namespaces are matched. | Valid selector | [selector](#selector) | |
```
Comment (emanic) [inline _includes/master/entityrule.md]: This is where an explanation in the introduction which we could cross-reference here would be useful. There are a lot of namespaces out there: Kubernetes, Linux, etc. It seems everyone has a namespace and so what namespace means in this context is not going to be clear. Right now, I think we have the info what is a Calico namespace in the intro of `GlobalNetworkPolicy` and `NetworkPolicy`...it's not something we can cross-reference.

Comment (caseydavenport) [reply _includes/master/entityrule.md]: Easy enough to say "selected kubernetes namespaces" here instead.

Comment (caseydavenport) [reply _includes/master/entityrule.md]: Would be good to have a page on namespaces and how they interact with diff orchestrators somewhere, but the calicoctl reference might not be the right place.

Comment (emanic) [reply _includes/master/entityrule.md]: Oh, so then these are kubernetes namespaces. Yeah, then you can just say "Kubernetes namespaces" that works fine.

**Comment by tmjd** [comment]
I don't think there is any reason to hold this PR anymore, the code is in the master components now.

**Comment by caseydavenport** [comment]
@emanic ready for an approval :)

**Pull request merged**